packages/@ama-sdk/core/.npmignore

```diff
@@ -0,0 +1 @@
+schematics/ng-add/mocks/
```

Comment:
I am fine with this change but just for my knowledge:
`dist/schematics/ng-add/mocks/` will not work without having to move this file? Or does it explicitly need to be beside or under the published package.json?

Reply:
As discussed, as we publish directly the dist folder it takes only npmignore present inside the dist folder